scripts/kani-regression.sh

```diff
@@ -38,6 +38,7 @@ TESTS=(
     "cargo-kani cargo-kani"
     "kani-docs cargo-kani"
     "kani-fixme kani-fixme"
+    "results-new-format results-new-format"
```

Comment:
Let's rename this to something like `reachability-checks` and add a note saying this is temporary.